processors/src/SvtRawDataProcessor.cxx

```diff
@@ -14,7 +14,7 @@ SvtRawDataProcessor::~SvtRawDataProcessor() {
 
 void SvtRawDataProcessor::initialize(TTree* tree) {
 
-    rawhits_   = new TClonesArray(Collections::RAW_SVT_HITS, 100000);  
+    rawhits_   = new TClonesArray("RawSvtHit", 100000);  
```

Comment:
Messy and confusing. We should fix this TClonesArray and std::vectors and what collections the event holds

Reply:
Switching to vectors should be done with another PR.